cmd/gossamer/export.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2019 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+
+package main
+
+import (
+	"os"
+	"path/filepath"
+
+	log "github.com/ChainSafe/log15"
+	"github.com/naoina/toml"
+	"github.com/urfave/cli"
+)
+
+// exportAction is the action for the "export" subcommand
+func exportAction(ctx *cli.Context) error {
```

Comment:
what is this? was this dumpConfig before?

Reply:
correct, it was `dumpConfig` before